tsconfig.json

```diff
@@ -3,19 +3,19 @@
     "allowSyntheticDefaultImports": false,
     "allowUnreachableCode": false,
     "allowUnusedLabels": false,
-    "alwaysStrict": true,
+//    "alwaysStrict": true,
```

Comment:
I am guessing that these settings will be re-enabled when you're done refactoring? We'll need to make sure that no other changes land while you're working through the refactoring PRs. Is that understanding correct?